packages/packages/api/src/devices/models/common/metadata.ts

```diff
@@ -1,7 +1,9 @@
 import { ProviderSource } from "./provider-source";
+import { SourceInfo } from "./source-info";
 
 export interface Metadata {
   date: string;
   source: ProviderSource;
+  data_source?: SourceInfo;
```

Comment:
So i thought the same thing but for other platfrom the source can be another app like Google or Apple so its not always a device

Reply:
This is an example https://dev.fitbit.com/build/reference/web-api/body/get-weight-log/

Reply:
Gotcha... yeah, not straightforward.

But I still think we should name something else... two "sources" on the same file is confusing, especially if one is `Fitbit` and the other `Withings`.

I'd rather have

```
{
  source: "Fitbit",
  device: {
    name: "Withings"
  }
}
```

than

```
{
  source: "Fitbit",
  data_source: {
    name: "Withings"
  }
}
```

Another option is to rename our `source` prop to `provider`, which actually might be the best one, but it'd be a breaking change.

But not a blocker, just trying to be careful with naming here as its a product interface, harder for change later.

Reply:
data_origin? Does this work?

Reply:
Thinking here, if we agree that `provider` would be the best name for the `source` prop, we could add it along side `source`, deprecate `source`, and keep `data_source` as you suggested... then, send a notification to DAPI cxs that it will be removed on the next major version.

So if you want to push as `data_source` that's fine... we can start the migration to `provider` - if agreed - in another PR.

Reply:
Yea i agree this makes sense heres the PR
https://github.com/metriport/metriport-internal/issues/743